I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 87, in script
    jump social_questions
ScriptError: could not find label 'social_questions'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 87, in script
    jump social_questions
  File "/Applications/renpy-8.3.3-sdk/renpy/ast.py", line 1699, in execute
    rv = renpy.game.script.lookup(target)
  File "/Applications/renpy-8.3.3-sdk/renpy/script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'social_questions'.

macOS-10.16-x86_64-i386-64bit x86_64
Ren'Py 8.3.3.24111502
Sustainopolis 1.0
Sat Dec  7 21:07:45 2024
